server/sandbox_run.go

```diff
@@ -74,9 +74,6 @@ func convertPortMappings(in []*pb.PortMapping) []*hostport.PortMapping {
 		if v.HostPort <= 0 {
 			continue
 		}
-		if v.Protocol != pb.Protocol_TCP && v.Protocol != pb.Protocol_UDP {
-			continue
-		}
```

Comment:
I know there are only 3 values defined in the map now, but is there any value to keeping this but adding Protocol_SCTP to guard against a future addition that might clonk CRI-O?  I lean towards no, that is this change is fine as is, but thought I'd ask just in case.

Reply:
I think if there's an expectation new map values succeed in CRI-O, I'd rather get clonked (and fix it) if they fail than skip them